R/tokenizers.R

```diff
@@ -39,60 +39,35 @@ NULL
 #'   stri_replace_all_regex stri_detect_fixed stri_replace_all_fixed
 #' @export
 tokenize_word <- function(x, split_hyphens = FALSE, verbose = quanteda_options("verbose")) {
-    named <- names(x)
+    
+    m <- names(x)
```

Comment:
This is more clear than `named`? 🙄

Reply:
I think it is common to use short names for variables with short life-span.

Reply:
😣